source/assets/js/modules/profile/services/vocab.service.js

```diff
@@ -247,8 +259,10 @@ angular.module('locApp.modules.profile.services').factory('Vocab', function($q,
                             returnNumber++;
 
                             // Set local storage once we have all the data.
+                            // The Value Data Type should use the Class list, not the literal property (datatypes) list
                             if(returnNumber >= listLength) {
-                                _setLocalStorage(resources, properties, datatypes);
+                                //_setLocalStorage(resources, properties, datatypes);
```

Comment:
Could you review this gist, https://gist.github.com/kirkhess/090bbc5010f31aea3ff08f2caf7f07d6

This test goes in create-property.test.js but on my system it only passes this test when it is at the top, all the other tests fail.  I was reluctant to commit this in because of this problem.

Reply:
Thanks Kirk; taking a look now.

Reply:
So to get the tests to pass, I moved your test `describe` function to the top level of the module and added some additional set-up calls in the `beforeAll` function. I also rebased to include a security fix and pushed up my changes in this [commit](6dcd2f0988d5ab689ed713e668f20999c5f33857). @ndushay could you do a final review and merge if you think the branch is ready?

I'm resolving this conversation.